### DISCUSSION TITLE
Playing mp3 over camera?

### DISCUSSION THREAD

**Discussion by Dvalin21** [blakeblackshear/frigate#5692] [Ask A Question]
Is it possible to have camera triggered by detection and then have mp3 played through the camera? Or possibly TTS? And if not, can this be an added feature? Thanks.

**Comment by blakeblackshear** [comment]
I don't know if there is an onvif standard for playing mp3 audio over a camera. If there is, maybe.

**Comment by Dvalin21** [comment]
Hope so. It would be a nice feature, but not anything necessary

**Comment by NickM-27** [comment]
This is already supported in go2rtc https://github.com/AlexxIT/go2rtc#stream-to-camera
